config/default/jenkins-infra.yaml

```diff
@@ -217,36 +170,48 @@ jenkins:
                   }
                 }
             - script: >
-                multibranchPipelineJob('docker-builder') {
-                  displayName "Docker Builder's Docker Build"
-                  branchSources {
-                    github {
-                      id('2020122201')
-                      scanCredentialsId('github-access-token')
-                      repoOwner('jenkins-infra')
-                      repository('docker-builder')
-                    }
-                  }
-                  factory {
-                    workflowBranchProjectFactory {
-                      scriptPath('Jenkinsfile_k8s')
+                [
+                  [repo: 'docker-packer', owner: 'jenkins-infra', name: 'Packer Docker Build', trust: 'TrustPermission'],
+                  [repo: 'docker-mirrorbits', owner: 'jenkins-infra', name: 'Mirrorbits Docker Build', trust: 'TrustPermission'],
+                  [repo: 'docker-datadog', owner: 'jenkins-infra', name: 'Datadog Docker Build', trust: 'TrustPermission'],
+                  [repo: 'docker-terraform', owner: 'jenkins-infra', name: 'Terraform Docker Build', trust: 'TrustPermission'],
+                  [repo: 'docker-builder', owner: 'jenkins-infra', name: "Docker Builder's Docker Build", trust: 'TrustPermission'],
+                  [repo: 'aws', owner: 'jenkins-infra', name: 'AWS Infra', trust: 'TrustNobody'],
```

Comment:
could you explain why this is trust nobody please?

Reply:
this repo is very much experimental atm, we'd like to change the permission level once we've worked out what we can safely run on a PR

Reply:
permission level doesn't have anything to do with what you can run though?

permission level dictates who is allowed to change a Jenkinsfile and have it validated.

everywhere else we use 'write access to the repo' as the who is allowed to change the file.

sure someone with write access _could_ change the file but i'm not sure this is really the right approach...

development also seems to be the wrong phase to add something like this...

Reply:
So keeping `TrustPermission` for all repos would be sufficient?

Reply:
yes, then I would question why it's configurable at all, it's not something that is normally changed, the default is fine in 99% of cases.